worlds/witness/options.py

```diff
@@ -50,10 +58,12 @@ class ShuffleLasers(Choice):
 
 
 class ShuffleDoors(Choice):
-    """If on, opening doors, moving bridges etc. will require a "key".
+    """
+    If on, opening doors, moving bridges etc. will require a "key".
     If set to "panels", the panel on the door will be locked until receiving its corresponding key.
     If set to "doors", the door will open immediately upon receiving its key. Door panels are added as location checks.

```

Comment:
I wonder if this line should be changed a bit, while it's still true that "door panels are added as location checks", it means a lot less now

Reply:
Could just go with "All"? And then maybe some sort of... "including ones that are usually not location checks?" Idk

Reply:
Maybe just "Some additional door panels" ?